core/src/test/kotlin/org/stellar/anchor/filter/Sep10JwtFilterTest.kt

```diff
@@ -22,7 +24,8 @@ import org.stellar.anchor.config.SecretConfig
 import org.stellar.anchor.filter.Sep10JwtFilter.APPLICATION_JSON_VALUE
 import org.stellar.anchor.filter.Sep10JwtFilter.JWT_TOKEN
 
-@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@Execution(SAME_THREAD)
+@Order(85)
```

Comment:
How do we decide the order when we add new tests?

Reply:
If the test is not long, don't assign the order. I made the slower test with higher priority. 

May be we can make it soemthing like `@Order(1000 - $seconds_to_run)`

Reply:
Are you seeing a noticeable improvement by ordering the tests? I think if we aren't seeing it, we should not be manually ordering them.

Reply:
It does improve if the thread count is slow.

Reply:
By default, the order should not be assigned unless it is taking long to run.